src/blocks/accordion/edit.js

```diff
@@ -53,22 +52,17 @@ const AccordionEdit = ( props ) => {
 	const { insertBlock } = useDispatch( 'core/block-editor' );
 
 	const {
-		getBlockAttributes,
 		getBlocksByClientId,
+		getBlockHierarchyRootClientId,
+		getSelectedBlockClientId,
+		getBlockAttributes,
 		isSelected,
-	} = useSelect( ( select ) => {
-		const blockEditorSelectors = select( 'core/block-editor' );
-
-		// Get clientID of the parent block.
-		const rootClientId = blockEditorSelectors.getBlockHierarchyRootClientId( clientId );
-		const selectedRootClientId = blockEditorSelectors.getBlockHierarchyRootClientId( blockEditorSelectors.getSelectedBlockClientId() );
+	} = useSelect( 'core/block-editor' );
 
-		return {
-			getBlockAttributes: blockEditorSelectors.getBlockAttributes,
-			getBlocksByClientId: blockEditorSelectors.getBlocksByClientId,
-			isSelected: isSelected || rootClientId === selectedRootClientId,
-		};
-	} );
+	// Get clientID of the parent block.
+	const rootClientId = getBlockHierarchyRootClientId( clientId );
+	const selectedRootClientId = getBlockHierarchyRootClientId( getSelectedBlockClientId() );
+	const isBlockSelected = isSelected || rootClientId === selectedRootClientId;
```

Comment:
For posterity: This change was needed because the way in which we had used `useSelect` previously, now errors out. This change allows the selector to function as expected.